### ISSUE TITLE
Frame at non root url breaks on pressing tx button

### ISSUE THREAD

**Issue by sammdec** [wevm/frog-archived#180]
### Describe the bug

I have 2 frames one is at the root url (https://caststorage-git-staging-sammdec.vercel.app/), it uses pages and frame co-location in the next.js app it resides in this works fine when testing in the warpcast frame validator.

However, I have created a new frame in the same app at a non root page (https://caststorage-git-staging-sammdec.vercel.app/bot), the frame validator can see the frame on page load and displays it correct however on pressing the subscibe button to trigger a tx request I receive a 500 error from frog with the following error message
```
Error: Invalid frame url: https://caststorage-git-staging-sammdec.vercel.app/bot. Expected: https://caststorage-git-staging-sammdec.vercel.app/api/bot-frame/subscribe.
```

### Link to Minimal Reproducible Example

https://caststorage-git-staging-sammdec.vercel.app/bot

### Steps To Reproduce

- Go to https://warpcast.com/~/developers/frames?url=https://caststorage-git-staging-sammdec.vercel.app/bot
- Click subscribe 
- See spinner and error shown

### Frog Version

0.7.4

### TypeScript Version

5.4.3

### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/frog/issues) for the bug I encountered.

**Comment by jxom** [comment]
Fixed

**Comment by sammdec** [comment]
Confirmed fixed after updating to 0.7.6
